### DISCUSSION TITLE
Incapsula Block Leading to 403

### DISCUSSION THREAD

**Discussion by ianzhg** [ercot/api-specs#26] [General]
Hi,

I wrote a piece of code to extract substation data from the server using API. Beginning today, all my requests have returned 403 forbidden. I checked the text message it seems Incapsula blocked my access. I tried using the simplest get public reports which does not require header configuration aside from authorization token and key but it is still returning 403.

I used the same authorization and key in postman and it works fine.

May I ask if this is you having a weekly rate limit and I hit that rate, or is it because my IP address is blocked?

Thanks,
Ian

**Comment by osoberanis-ercot** [comment]
Hello Ian,

Can you give us more information on the request and response? Could you please post it here so that we can troubleshoot it. 

Can you also give me your system username? I want to see if I can troubleshoot it based on your username.

One more thing I'm thinking are you sure you not storing the bearer token in some kind of variable or file?  

Thank you,

Omar
